construdelas/Thais_Bighetti/Modificando_Json/Exercicio_Thais_Bighetti.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+"""
+1 - Leiam o arquivo json transformando o em um dicionário
+2 - classificação dependente do gênero:
+mulheres: 1,50 - 1,55 - Estatura baixa
+1,56 - 1,70 - Estatura mediana
+1,71 - 1,89 - Alta estatura
+-----------------------------------------------------------------------------------
+Homens: 1,50 - 1,60 - Baixa estatura
+1,61 - 1,75 - estatura mediana
+1,76 - 1,89 - Alta estatura
+-----------------------------------------------------------------------------------
+3 - Se a pessoa obeteve a maior potuação entre as pessoas da lista
+4 - a lista pontuacao
+5 - a média da pontuacao
+6 - Nome da pessoa
+-----------------------------------------------------------------------------------
+gerar um novo json:
+{ 'Nome Completo': 'Chica da Silva',
+'Estatura': 'Baixa' | 'Média' | 'Alta',
+'Maior Pontuacao': True | False,
+'Pontuacao': [13, 7, 29, 4, 1, 11, 2, 22],
+'Media da Pontuacao': 11,125
+}
+"""
+
+arquivo = open("../../../../Jason/exercicios.json", "r")
+dicio = json.load(arquivo)
+
+for pessoas in dicio['Pessoas'] :
```

Comment:
[suggestion] O que você acha de trabalhar um pouco mais no nome das variáveis? 
Este loop `for` irá pegar uma `pessoa` dentro da lista de `Pessoas` oriunda do arquivo.
Do modo atual fica um pouco confuso, pois a variável `pessoas` remete algo que aparenta ser uma lista/múltiplos objetos ao invés de um objeto só.